### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by AyoMoses1**
# Feature Implemented: Styling
I implemented the styling part of this project. Please feel free to review and drop your feedbacks where necessary

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Navbar = () => {
           {links.map((link) => (
             <React.Fragment key={link.text}>
               <li>
-                <NavLink to={link.path}>{link.text}</NavLink>
+                <NavLink to={link.path} className={styles.link}>{link.text}</NavLink>
               </li>
```
Comment (TadesseDev) [inline src/components/Navbar.js:24]: ### Impressive job @AyoMoses1, I just have a couple of suggestions to make, 
- [ ] As you can see from Zeplin design there is a user Icon, which is missing from your project. Please add this icon with the appropriate styling. This is to make sure you have the exact copy of the design. 
- [ ] Also there is a vertical line between the `comment, Remove, edit` texts, which are under the book title, Besides that, Notice only some of the text is blurred than the others and only the book title is bold. Please take a look and make the improvements. 

### Zeplin design
Screen Shot 2023-03-10 at 5 08 16 AM
### Your design 
Screen Shot 2023-03-10 at 5 10 27 AM

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @AyoMoses1 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitFlow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.js (hunk 1 above)
Comment (AyoMoses1) [reply src/components/Navbar.js:24]: Thanks so much @TadesseDev . It was an oversight from my part. I really appreciate

Comment (TadesseDev) [reply src/components/Navbar.js:24]: Your doing great @AyoMoses1 ✅

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @AyoMoses1, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**